src/CommonITILValidation.php

```diff
@@ -701,6 +701,23 @@ public static function getStatusColor($value)
         return $style;
     }
 
+    /**
+     * Get Ticket validation approver link
+     *
+     * @param string    $itemtype
+     * @param integer   $items_id
+     **/
+    public static function getApprover($itemtype, $items_id)
+    {
+        if (empty($items_id)) {
+            return null;
+        }
+
+        $approver = $itemtype::getById($items_id);
+
+        return $approver->getLink();
```

Comment:
IMHO, this method should return a `CommonDBTM` instance or a `null` value. Indeed, one may want to use the result of this method to display something else than a link.

By the way, I wonder if this method is really useful. The following code snippet, usable in the search class itself would probably do the same:
```php
if (is_a($itemtype, CommonDBTM::class, true) && ($approver = $itemtype::getById($items_id)) !== null) {
    // $approver->getLink();
}
```